test/test_team.py

```diff
@@ -767,3 +767,96 @@ def speaking_team(bot, state):
     # check that we finished without problem
     assert state['round'] == 1
     assert state['turn'] == 3
+
+
+def test_valid_paint_background():
+    test_layout = """
+        ##################
+        #.#... .##.     y#
+        # # #  .  .### #x#
+        # ####.   .      #
+        #      .   .#### #
+        #a# ###.  .  # # #
+        #b     .##. ...#.#
+        ##################
+    """
+    overlay = [
+               ((0,0) , "#AABBCC"),
+               ((1,1) , "#BBCCDD"),
+               ((100, 200) , "#DDCCAA"), # silently ignore coords out of the maze
+               ]
+
+    parsed = parse_layout(test_layout)
+
+    def overlay_team(bot, state):
+        for pos, color in overlay:
+            bot.paint_background(pos, color=color)
+        return bot.position
+
+    state = setup_game([overlay_team, overlay_team], max_rounds=2, layout_dict=parsed, raise_bot_exceptions=True)
+
+    overlay_in_maze = [{'pos':coord, 'color':color} for coord, color in overlay if coord[0] < 32]
```

Comment:
Idk, but for simplicity I guess we could just write it out for the two entries? :)